lib/engine-addon.js

```diff
@@ -143,10 +144,12 @@ var buildVendorJSTree = memoize(function buildVendorJSTree(vendorTree) {
 
 var buildVendorCSSTree = memoize(function buildVendorCSSTree(vendorTree) {
   // Filter out the CSS so that we can process it correctly.
-  return new Funnel(vendorTree, {
+  var vendorCSSTree = new Funnel(vendorTree, {
     include: ['**/*.css'],
     exclude: ['vendor/**/*.*']
   });
+
+  return new BroccoliDebug(vendorCSSTree, this._debugLabel + ':vendor-style:input');
```

Comment:
Minor suggestion: Can we create a helper function for constructing these labels? Maybe something like `this._debugLabel('vendor-style:input');` and change the current `_debugLabel` to be `_debugLabelPrefix` or something?

It's not 100% necessary, but I feel it would be nicer and makes maintainability better in the long run (e.g. if we want to change the format or the labels).